### PR TITLE
feat: few bug fixes (cursor pointer, arc integration, alerts components)

### PR DESCRIPTION
**Pull request by tcheee**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,56 +1,120 @@
...
+      gtag('js', new Date());
+      gtag('config', '%VITE_GOOGLE_ANALYTICS_TRACKING_ID%');
+    </script>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <img
+      src="https://jumper.exchange/static/media/logo.gif"
+      referrerpolicy="no-referrer-when-downgrade"
+      style="opacity: 0"
+    />
```
Comment (dennyscode) [inline index.html:117]: What´s happening here?

File: index.html
Comment (dennyscode) [inline index.html:1]: Is there substiantial changes going on here? Or is it mostly linting?

File: src/components/Widgets/Widgets.tsx [hunk 2]
```diff
@@ -81,11 +88,6 @@ export function Widgets() {
 
   return (
     <>
-      {import.meta.env.MODE === 'testnet' && (
-        <Grid item xs={12} mt={theme.spacing(3)}>
-          <TestnetAlert />
-        </Grid>
-      )}
```
Comment (dennyscode) [inline src/components/Widgets/Widgets.tsx:88]: Testnet Messagebox removed in favor of the InfoAlert?

File: src/components/Alerts/AlertContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+  if (solanaSelected) {
+    return <SolanaAlert />;
+  } else if (isMultisig) {
+    return <MultisigSideAlert />;
+  } else if (import.meta.env.MODE === 'testnet') {
+    return <TestnetSideAlert />;
+  } else {
+    return null;
+  }
```
Comment (dennyscode) [inline src/components/Alerts/AlertContainer.tsx:34]: Edge case: What´s happening when you are on the testnet with a multisig-wallet and having Solana selected? Is it meant to be that we´ll always just end up showing one of these?

In last design call, we agreed to keep the current "MessageBox" for testnet. How come we are moving this down?

File: .env.development [hunk 4]
```diff
@@ -13,7 +13,4 @@ VITE_STRAPI_API_TOKEN=2350647febb39fe14dea85ee80e0f90384266c8dce548cf7d1d8190159
...
...JQyBLRVktLS0tLVxyXG5NSUlDSWpBTkJna3Foa2lHOXcwQkFRRUZBQU9DQWc4QU1JSUNDZ0tDQWdFQXZDUTBsT2tIQnp0aUMvNW1TbnlrXHJcbnVBUzA5QmxlK25DM2hlQ051b1RHaEo3Z0tkTEw0QXlKMGNORUNLWGsydkhKTjBJeHlYOXVid1hpa3BteFhXMXNcclxuTzliUk5HRHVuSTUyQ29ac2R3aVNON3g0Q01SMzlhT1l4THVmcVplYVprUWdUU1pvZldBYTZnMU5LRzhwSUczMlxyXG5iaUhaSjNFVkZZSnFHOERJR2xPNExhbk1KNGxIZUtUdXEvb1NHMmt3d3B3NDdDbWNSVXpvbzlLbHpYVkJUNUE1XHJcbkV4SHI4bUJPekV0NDcvenlqTG1VVGpRVVZLdWJ2RFlzcVR0Mzd6VkhCNm5BS2ZiYmhybnBPU3U5R3RGcXFsdVRcclxuS1ZhV1hnMFc5SXM3S0dZYm13dFRVcXlGbnl0WVdXY0dKSCtYaU9UQnh5MjJEcTJsNjRaa3Jlc0FKci81Sm50SVxyXG5HMUVHWXdyeFJ1dkRKbTBTRXIzNEY4ZUxETnNPNm42SldoMFd1NzVIbytPcUkvY3pEUGk1V2NnTmlDaWZTcndzXHJcbktZL1RCRS9rT2grQ2FRTTBYcWIrYWErL3U4eURlalJGREE4bHNxSE9BeFUvYXZXV0ZsWktDcUhweThQUm93MGNcclxudHBWSmFtTVZoY2ZKZUN0dm5RdkE2dnVXZzZ3emdvY2s5ZGp5Y0hNbXdRbEg0bWEvQXZnNGF2YzlLN0Z0UlNYL1xyXG5SQTFCQkpieGp3Ylk3ZlowQXpaR2kzMWhTa0pOLytXSU9BZHVmZmM2M0VOL0JGRjhxbjAxMSsrNnNBUU41MDF4XHJcblA3enV5K3dITjJmdVZGYmY1NkZpd3NEa1I1dU5QWmtxK2NyZGVnUm5MNUZwbjNnTFZqWXVLaDBsVEExS3BnYzVcclxudkM3K2o1SERlWmk4QUU2Y1JVUGV4U1VDQXdFQUFRPT1cclxuLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0tXHJcbiJ9LCJ0eXBlIjoiYXV0aF90b2tlbiIsImlhdCI6MTcwNzkyNDg3MH0.OY1EKDcAJLWKKlIIMmeE9T8UCIk3j9JX5I8LFQKJ1hE
```
Comment (dennyscode) [inline .env.development:18]: How come we can delete this on development but not across other env´s?

File: src/hooks/useAccounts.ts [hunk 5]
```diff
@@ -145,7 +145,7 @@ export const useAccountConnect = () => {
         walletName: combinedWallet.evm.name,
         chainType: ChainType.EVM,
         chainId: await combinedWallet.evm.getChainId(),
-        address: `${await combinedWallet.evm.getAccounts()}`,
+        address: `${(await combinedWallet.evm.getAccounts())[0]}`,
```
Comment (dennyscode) [inline src/hooks/useAccounts.ts:148]: Nice! We might also adjust that part for the user tracking in regards of arcx. They stated that multiple wallets are sent.

File: src/components/Alerts/SolanaAlert.tsx [hunk 6]
```diff
@@ -6,19 +6,10 @@ import { useChainTokenSelectionStore } from 'src/stores';
...
-    const isSelected =
-      sourceChainToken?.chainId === ChainId.SOL ||
-      destinationChainToken?.chainId === ChainId.SOL;
-    return isSelected;
-  }, [destinationChainToken, sourceChainToken]);
 
   return (
     <InfoAlert
-      active={solanaSelected}
+      active={true}
```
Comment (dennyscode) [inline src/components/Alerts/SolanaAlert.tsx:12]: I don´t get the whole picture, yet. But if we went on like that, you might as well remove the "active" prop at all. But let´s have a chat.

**Review by dennyscode — CHANGES_REQUESTED** [review]

**Review by Addminus — DISMISSED** [review]

File: index.html [hunk 7]
```diff
@@ -1,56 +1,120 @@
...
+      href="/favicon.svg"
+      sizes="any"
+      media="(prefers-color-scheme: light)"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      href="/favicon.png"
+      media="(prefers-color-scheme: light)"
+    />
```
Comment (chybisov) [inline index.html:58]: Are we sure this is better formatted? 😄

**Pull request closed without merging**